train_controlnet3d.py

```diff
@@ -771,6 +769,7 @@ def load_model_hook(models, input_dir):
     vae.to(accelerator.device, dtype=weight_dtype)
     unet.to(accelerator.device, dtype=weight_dtype)
     text_encoder.to(accelerator.device, dtype=weight_dtype)
+    controlnet.to(accelerator.device, dtype=weight_dtype)
```

Comment:
This should happen automatically. I just added it in my branch because I disabled it in prepare but it didn't help so I would not worry about it.

Reply:
Ah ok, I'll remove it then